lncfg/sweeper.go

```diff
@@ -3,11 +3,24 @@ package lncfg
 import (
 	"fmt"
 	"time"
+
+	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
+)
+
+const (
+	// MaxFeeRateFloor is the smallest config value allowed for the max fee
+	// rate in sat/vb.
+	MaxFeeRateFloor = 100
```

Comment:
Should we use typed constants here to avoid them being used elsewhere with a different  unit/type?

Reply:
yeah I like it, updated.